taxi/src/test/java/com/epam/training/taxi/CalculatorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.training.taxi;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CalculatorTest {
```

Comment:
This test class is 💯 % suspicious.  
- The application should be covered by unit tests but there are only commented out unit tests in this class.
- Unit tests should be public.
- Should use the suggested naming conventions 'test[TestedMethod]Should[DoSomething]When[Condition]()'